Content.Server/Genetics/Systems/GenomeSystem.cs

```diff
@@ -0,0 +1,234 @@
+using Content.Server.Genetics.Components;
+using Content.Shared.GameTicking;
+using Content.Shared.Genetics;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+using Robust.Shared.Reflection;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Content.Server.Genetics.Systems;
+
+/// <summary>
+/// Assigns each <see cref="GenomePrototype"/> a random <see cref="GenomeLayout"/> roundstart.
+/// </summary>
+public sealed class GenomeSystem : EntitySystem
+{
+    [Dependency] private readonly IPrototypeManager _proto = default!;
+    [Dependency] private readonly IReflectionManager _reflection = default!;
+    [Dependency] private readonly IRobustRandom _random = default!;
+
+    // This is where all the genome layouts are stored.
+    // TODO: store on round entity when thats done, so persistence reloading doesnt scramble genes
+    [ViewVariables]
+    private readonly Dictionary<string, GenomeLayout> _layouts = new();
+
```

Comment:
This is going to break on prototype reloading and save/load so not really sure why this is here.

Reply:
its what stores the rounds random layout, like wires

Reply:
That's still gonna break tho.

Reply:
round entities + nullspace entity saving wyci